src/components/virtualRepeat/virtual-repeater.js

```diff
@@ -100,6 +102,25 @@ function VirtualRepeatContainerController($$rAF, $scope, $element, $attrs) {
   /** @type {number} Amount to offset the total scroll size by. */
   this.offsetSize = parseInt(this.$attrs.mdOffsetSize, 10) || 0;
 
+  if(this.$attrs.hasOwnProperty('mdTopIndex')) {
+    /** @type {function} An Angular expression that binds to topIndex on the scope */
+    this.bindTopIndex = $parse(this.$attrs.mdTopIndex);
+    /** @type {number} The index of the item that is at the top of the scroll container */
+    this.topIndex = this.bindTopIndex(this.$scope);
+
+    if(!angular.isDefined(this.topIndex)) {
+      this.topIndex = 0;
+      this.bindTopIndex.assign(this.$scope, 0);
```

Comment:
It took some research on StackOverflow to figure out because this directive won't work with isolate scope binding.